src/ServicePulse.Host/vue/src/router/index.ts

```diff
@@ -1,22 +1,33 @@
 import { createRouter, createWebHashHistory, type RouteRecordRaw } from "vue-router";
-import config from "./config";
+import config, { type RouteItem } from "./config";
 
 function meta(item: { title: string }) {
   return { title: `${item.title} • ServicePulse` };
 }
 
-const routes = config.map((item) => {
-  return {
+const routes = config.flatMap<RouteRecordRaw>((item) => {
+  const result: RouteRecordRaw[] = [];
+  result.push({
     path: item.path,
+    name: item.path,
```

Comment:
I added a name to the routes to support the lookup

Reply:
Is this supposed to be the `.path` as well? This is replicating the `path` above.